src/NzbDrone.Core/Download/DownloadClientBase.cs

```diff
@@ -147,5 +147,7 @@ protected ValidationFailure TestFolder(string folder, string propertyName, bool
 
             return null;
         }
+
+        public virtual void MarkItemAsImported(DownloadClientItem downloadClientItem) { }
```

Comment:
Maybe we should add a _logger.Debug here to indicate that this client does not have a Post-Import action?